client/src/routes/WorkspaceRoute/Workspace.js

```diff
@@ -65,9 +66,60 @@ const move = (source, destination, droppableSource, droppableDestination) => {
   return result;
 };
 
+const AudioState = (wsMusic) => {
+  let playAudioIndex = [0, 0, 0, 0];
+  const [audioIndex, setAudioIndex] = useState(playAudioIndex);
+  const [audioPlaying, setAudioPlaying] = useState(false);
+
+  const audioToggle = () => {
+    if (audioPlaying) {
+      setAudioPlaying(false);
+      for (let i = 0; i < wsMusic.length; i++) {
+        if (wsMusic[i][audioIndex[i]] != null) {
+          wsMusic[i][audioIndex[i]]["audio"].pause();
+        }
+      }
+    } else {
+      setAudioPlaying(true);
+      // for (let i = 0; i < wsMusic.length; i++) {
+      //   if (wsMusic[i][audioIndex[i]] != null) {
+      //     wsMusic[i][audioIndex[i]]["audio"].play();
+      //   }
+      // }

```

Comment:
Can you delete these comments?

Reply:
For now, I'll keep these comments since not all of the audio is implemented yet and these lines are part of that. 👍🏽